api/definitions.yaml

```diff
@@ -79,17 +81,23 @@ NewMessage:
 MessageStatusValue:
   type: string
   x-extensible-enum:
-    - PROCESSING
     - ACCEPTED
     - FAILED
+    - PROCESSED
```

Comment:
we need to document a state diagram somewhere that describes the initial and final states and their meanings